.gitignore

```diff
@@ -9,6 +9,9 @@ doc/_build/
 src/pyq/version.py
 html/
 
+.venv*/
+venv*/
```

Comment:
this doesn't belong here.

Reply:
Would you rather a separate PR just for this change? The `.gitignore` should exclude Python virtual environments in some capacity.

Reply:
I don't believe venvs belong in code repository. For example, I keep my venvs in the `.virtualenvs` directory. Adding all possible locations will grow `.gitignore` dramatically and this is not necessary.

Reply:
Per-project virtual environments are a widespread practice. You'll find most large Python projects have a few lines in their `.gitignore` to exclude them. Besides, we don't need to add "all possible locations" - no need to let perfect be the enemy of good. This addition takes care of all of the most common ones in just two lines.